doc/node-types-html/index.html.in

```diff
@@ -15,10 +15,225 @@
+                thumb.on('click',onThumbExampleClick);
+                info.append(thumb);
+
+                if(i==0){
+                    /* load the first example */
+                    loadExampleImage(thumb);
+                }
+            }
+            // Added hidden textarea element to store code
+            var hidden_code = $('<textarea></textarea>');
+            hidden_code.appendTo('#example-info');
+            hidden_code.attr('id','example-info-code');
+            hidden_code.disabled=true;
+            /* adding the get-code button */
+            var getCodeButton = $('<img></img>');
+            getCodeButton.attr('src','images/button_get_code.png');
+            getCodeButton.attr('id','example-get-code');
+            info.append(getCodeButton);
+            var code = $('#example-image').data('code'); 
+            $.get(String(code),function(data){              //Getting code from path
+                hidden_code.html(String(data));
+            }); 
+            getCodeButton.on('click',copyExampleCode);
+            }
+        }
+
+        /* handles the click on a thumbnail */
+        function onThumbExampleClick(event){
+            loadExampleImage($(this));
+        }
+
+        /* loads the correspondent image from the thumbnail */
+        function loadExampleImage(_thumb)
+        {
+            /* removing old image reference */
+            $('#example-image-container').empty();
+            $("img[id^=thumb_]").each(function(){
+                $(this).removeClass('example-thumb-active');
+            });
+
+            /* loading selected image */
+            var imageURL = $(_thumb).data('large');
+
+            // SVG Loading happens here 
+            var image = $("<img></img");
+            image.attr("src",imageURL);
+            image.attr('id','example-image');
+            image.data('code',$(_thumb).data('code'));
+
+            /* adding the loaded image into the drag view */
+            $("#example-image-container").append(image);
+
+            /* marking the active thumb */
+            $(_thumb).addClass('example-thumb-active');
+
+            /* making the image draggable */
+            image.draggable();
+
+            /* removing existing copy feedback and restoring the copy button */
+            if($('#example-info').has('#example-code-copied')){
+                $('#example-code-copied').remove();
+                $('#example-get-code').show();
+            }
+        }
+
+        /* --------------------------------------------------------
+         * COPY EXAMPLE CODE
+         * Implement the algorithm to read the example
+         * code here. The var code already stores url 
+         * when creating the Entry in the main loop
+         * --------------------------------------------------------
+         */
+        function copyExampleCode(event){
+            var x = document.getElementById('example-info-code');
+            selecttext(x);
+            y=copycode();
```

Comment:
Split the returned var from the function. 

```y =  copycode();```

Also where do you declare this var ? Why not using it to verify if the copycode was successful ?